docs/build/lotus/lotus-lite.md

```diff
@@ -34,27 +57,46 @@ make clean all
 sudo make install
 ```
 
-3. On the lite-node, create an environment variable called `FULLNODE_API_INFO` and give it the following value while calling `lotus daemon --lite`:
+1. On your lite-node clone the [Lotus GitHub repository](https://github.com/filecoin-project/lotus) and create the executable, but do not run anything yet:
 
-```
-$FULLNODE_API_INFO=/ip4/YOUR_FULL_NODE_IP_ADDRESS/tcp/28001 lotus daemon --lite
+    ```shell
+    git clone https://github.com/filecoin-project/lotus
+    cd lotus
+    make clean all
+    sudo make install
+    ```
 
-> 2021-03-02T23:59:50.609Z        INFO    main    lotus/daemon.go:201     lotus repo: /root/.lotus
-> ...
-```
+    If you run into errors here, it may be because you don't have all the Lotus dependencies installed. Take a quick look at the [Lotus Getting Started guide](/get-started/lotus/installation/#software-dependencies) and double check that you have all the dependencies installed, along with Golang and Rust.
 
-Make sure to replace `YOUR_FULL_NODE_IP_ADDRESS` with the IP address of your full-node.
+## Start the lite-node
 
-4. You can now interact with your Lotus lite-node:
+You've got the Lotus executables ready to go, and you have access to a Lotus full-node. All that's left is connecting your Lotus lite-node to the full-node!
 
-```
-lotus wallet balance f10...
+1. On the lite-node, create an environment variable called `FULLNODE_API_INFO` and give it the following value while calling `lotus daemon --lite`:
 
-> 100 FIL
-```
+    ```shell
+    FULLNODE_API_INFO=/ip4/YOUR_FULL_NODE_IP_ADDRESS/tcp/28001 lotus daemon --lite
```

Comment:
`28001` is not a lotus standard port. It's just something I personally use in aws. The default port is `1234`

Reply:
Ah cool, thanks for the heads up! Just updated things now.